src/averageAges.js

```diff
@@ -20,6 +20,29 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const totalAge = people.map(human => human.sex === 'm'
```

Comment:
Instead of calculating both cases and only at the end of the function returning the correct case, first of all, better to filter `people` based on the second argument (use ternary inside filter callback).
For example, in the current function, filter people by `sex` and if `century` is defined - by century (when the person died) as well.

Reply:
done